app.js

```diff
@@ -67,25 +79,34 @@ const App = React.createClass({
               window.localStorage.setItem('listenerName', e.target.value)
             }
           }),
+          React.DOM.input({
+            className: 'checkbox',
+            type: 'checkbox',
+            defaultChecked: self.state.autoSend,
```

Comment:
it defaults to the last state that was set which is stored in localStorage -> the setting is persisted and picked up again on the next launch.
If nothing is stored in localStorage, it defaults to `false`. See app.js line 15:
https://github.com/tkothe/playing/blob/6dad30b2d5333fa7a80181c6560f71d52273145d/app.js#L15

So if we want to default this to `true`, that's where it should be changed imho.
What do you think?